### ISSUE TITLE
Implicit multiplication not working!?

### ISSUE THREAD

**Issue by COALROCK8642** [Qalculate/qalculate-qt#89]
Why is this not working? May be we need guide to settings.

![image](https://github.com/Qalculate/qalculate-qt/assets/88789897/6b15474a-a34d-4926-84b3-a12198e684be)

**Comment by hanna-kn** [comment]
What is it that does not work as you expect?

The only odd thing I notice in the screenshot is that min decimals seems to be set to 10.

The interpretation of `1/2x` is determined by "parsing mode" setting (under Parsing & Calculation in preferences).

**Comment by COALROCK8642** [comment]
But what will I do if my expression is `2pi/2*4*root(2,2)` I expect this to be `2*pi/8*root(2,2)` and it comes `((2 × pi)/2) × 4 × root(2, 2)`

![image](https://github.com/Qalculate/qalculate-qt/assets/88789897/75ca64aa-25bc-4771-8108-e10b80e4dfa7)

**Comment by COALROCK8642** [comment]
Yes! Yes! Yes! Exact words. Within brackets (if any) all multiplications will have presedence and then divisions will happen.

**Comment by hanna-kn** [comment]
This would not be difficult to implement, but it seems a bit unusual. Is explicit multiplication performed before division anywhere else?

**Comment by COALROCK8642** [comment]
> This would not be difficult to implement, but it seems a bit unusual. Is explicit multiplication performed before division anywhere else?

But this is nesscary when people will solve compound fractions.

**Comment by COALROCK8642** [comment]
for example, `(x1*x2/y1*y2)/z1*z2` can be just written `x1*x2/y1*y2//z1*z2` and will process like, `((x1*x2)/(y1*y2))/(z1*z2)` and `/,//,///` will scale according to compound fractions.

**Comment by hanna-kn** [comment]
My question remains. Is this notation and order of operations used anywhere?

It seems safer to use parentheses instead, and it does remove the necessity of parentheses for expressions with addition or subtraction in the numerator or denominator (e.g. `(x1+x2)/(y1+y2)`).

**Comment by SanderBouwhuis** [comment]
Isn't the default rule that multiplication and division are equal priority, and will be evaluated left-to-right?

Here is a sensible list how it 
![C++ Operator Precedence](https://github.com/user-attachments/assets/3e0757a1-dae3-4070-b77e-57d14575fafa)
works in C++:

**Comment by hanna-kn** [comment]
> Isn't the default rule that multiplication and division are equal priority, and will be evaluated left-to-right?

Yes, it is. You can see the order of operations used in Qalculate at https://qalculate.github.io/manual/qalculate-expressions.html#qalculate-expression-operators (in the list below the table).

The creator of this issue requested a different unusual order of operations, and/or special operators for different levels of divisions.
